### PR TITLE
create python 3.7 wheel for 1.8.2

### PR DESCRIPTION
**Pull request by plasmax**
Provides a github actions build system for a Python 3.7 Wheel containing all dependencies (except python itself).

Based off https://github.com/alembic/alembic/pull/335 for Python 2.7 Wheel.

Uses Docker with the aswf/ci-usd:2021.6 image and Imath v3.0.4.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -58,19 +100,17 @@ def build_extension(self, ext):
...
+        subprocess.check_call(
+            [cmake_command, ext.sourcedir] + cmake_args, cwd=self.build_temp, env=env)
+        subprocess.check_call(
+            [cmake_command, '--build', '.'] + build_args, cwd=self.build_temp)
+
+
 setup(
     name='alembic',
-    version='1.7.16',
+    version='1.8.2',
```
Comment (manneohrstrom) [inline setup.py:113]: this should most likely not be hard coded but should contain the current tag name for tags and the short hash for commits? e.g. something like https://stackoverflow.com/questions/58177786/get-the-current-pushed-tag-in-github-actions

**Review by manneohrstrom — CHANGES_REQUESTED** [review]
this looks good - i don't have a lot of helpful comments. I suggest the following:

- We need a single PR which includes 2.x and 3.x
- let's do a super quick CR of that and then merge that into master
- send a PR back to the main alembic repo.

Let's get these various branches closed out and work from master.

File: setup.py (hunk 1 above)
Comment (plasmax) [reply setup.py:113]: Thanks Manne - I will open a new PR with both python2.7 and python3.7 manylinux wheel builds.

As pyalembic can also be built from source locally, I'm unsure of the best approach towards automated versioning. Additionally, github actions currently doesn't support a simple way to pass build-args to Docker, which would need access to these environment variables.

Currently, the values are hard coded in the CMakeLists.txt files - maybe we could extract the version from there?

**Comment by plasmax** [comment]
Closing this PR in favour of framestore/alembic#2

**Pull request closed without merging**